frontend/src/App.js

```diff
@@ -1,9 +1,32 @@
-import React from 'react'
+import React from 'react';
+import { user } from './reducers/user';
+import { BrowserRouter, Switch, Route } from 'react-router-dom';
+import { configureStore, combineReducers } from "@reduxjs/toolkit";
+import { Provider } from "react-redux";
+import { StartPage } from './pages/StartPage';
+import { Secret } from './pages/Secret';
+
+const reducer = combineReducers({
+  user: user.reducer
+})
+
+const store = configureStore({ reducer })
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+    <Provider store={store}>
+      <BrowserRouter>
+        <main>
+          <Switch>
+            <Route path='/' exact>
+              <StartPage />
+            </Route>
+            <Route path='/secrets' exact>
+              <Secret />
+            </Route>
+          </Switch>
+        </main>
+      </BrowserRouter>
```

Comment:
Since you're using react router it would've been nice if signup and login were on different routes as well :)